cli/Cargo.toml

```diff
@@ -49,6 +49,7 @@ pest.workspace = true
 pest_derive.workspace = true
 regex.workspace = true
 rpassword.workspace = true
+scm-record = "0.1.0"
```

Comment:
Obvious nit: move to a workspace dependency. I really hope `cargo add` supports workspace dependencies sometime in the future...